terraform/main.tf

```diff
@@ -1,3 +1,4 @@
 module "rds" {
   source = "./modules/rds"
-}
\ No newline at end of file
+}
+
```

Comment:
Probably it was used for testing. Probably, it can be removed.

Reply:
Hey Guys, just clarify i am following [terraform]( https://www.terraform.io/docs/modules/index.html), where advises that "main.tf, variables.tf, outputs.tf. These are the recommended filenames for a minimal module, even if they're empty. main.tf should be the primary entrypoint. For a simple module, this may be where all the resources are created. For a complex module, resource creation may be split into multiple files but any nested module calls should be in the main file."

Reply:
Hi @mfreitassm, thanks for clarifying it. :)

The files you mentioned should be in the module folder, so for this PR there is no need for a "terraform/main.tf" file.
Would it be great if you add a README.md file to document your module.

I'd suggest us to follow the deliverables in  this [PR](https://github.com/devopsacademyau/2020-jun-project1-group1/pull/19/commits/f8039bf77d0d5442bb5b2f4eb144d2d3f9fc5fd4)

```
terraform/modules/container_registry/README.md
terraform/modules/container_registry/main.tf
terraform/modules/container_registry/outputs.tf
terraform/modules/container_registry/variables.tf
```

Reply:
Hey @mfreitassm, I understood that, but since that we have multiple people working with terraform modules. If you add those files, it might cause merge conflict with your peer PR 🙄

Once that we start to put everything together, have all modules working, we can create those files. 😄

Reply:
Hi @fabioacp, Thanks for reply!

Actually I think there is no need to create a module only for the SSM parameter in my PR because we'll only have 4 single parameters and all of them is related with RDS itself. Said so, my suggestion is to put it inside the RDS module (in `modules/rds/main.tf`), what you guys think?

About `terraform/main.tf`, i understood this is the "main file" or entry point to the whole project and all module calls should be in that file. So I called the rds module on this file. This file is not inside of any module it is in terraform folder (root directory for our terraform code).
 
I've just deleted the output.tf due to another request because the file was empty.

Reply:
ok, np.